packages/shared/src/hooks/profile/useDevCard.ts

```diff
@@ -21,7 +21,7 @@ export interface DevCardData {
   sources: Source[];
 }
 
-interface UseDevCard {
+export interface UseDevCard {
```

Comment:
If you decide to follow my suggestion above, we no longer need to export this type.

Reply:
I think we can keep this, I used `Pick` on this type for the props.